Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -2850,7 +2854,26 @@ as modifying the License.
 
     <!-- XBUT: submission result dialog close button -->
     <string name="submission_result_button_dialog_close">OK</string>
+    <!-- ####################################
+              Heading Fragments
+        ###################################### -->
+    <string name="heading_submission_contact">TAN Anfrage per Telefonanruf</string>
+    <string name="heading_submission_dispatcher">Welche Informationen liegen ihnen vor?</string>
+    <string name="heading_submission_intro">Wurden Sie getestet? So funktioniert das Corona-Warn-System</string>
+    <string name="heading_submission_tan">TAN Eingabe</string>
+    <string name="heading_main">Hauptseite der Corona-Warn App</string>
+    <string name="heading_onboarding_privacy">Einführung Seite 2 von 5. Datenschutz</string>
+    <string name="heading_onboarding">Einführung Seite 1 von 5. Gemeinsam Corona bekämpfen</string>
+    <string name="heading_onboarding_tracing">Einführung Seite 3 von 5. Wie Sie die Risiko Ermittlung ermöglichen</string>
+    <string name="heading_onboarding_test">Einführung Seite 4 von 5. Falls Sie positiv getestet wurden</string>
+    <string name="heading_onboarding_notifications">Einführung Seite 5 von 5. Warnungen erhalten und Risiken erkennen</string>
+    <string name="heading_risk_details">Risiko Ermittlung</string>
 
+    <!-- ####################################
+          Button Tooltips
+    ###################################### -->
+    <string name="back_button">Zurück zur vorherigen Seite</string>
```

Comment:
we name the stuff always in this order: `view_description` so this one should be `button_back`

Reply:
please also add an comment for our translators